Makefile

```diff
@@ -12,6 +12,7 @@ ifneq ($(GRAPL_RUST_ENV_FILE),)
 DOCKER_BUILDX_BAKE_OPTS += --set *.secrets=id=rust_env,src="$(GRAPL_RUST_ENV_FILE)"
 endif
 COMPOSE_IGNORE_ORPHANS=1
+COMPOSE_PROJECT_NAME ?= grapl
```

Comment:
Even though it's a no-op, this is important to leave in, mainly because of https://github.com/grapl-security/grapl/pull/696/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R102

(Basically, localstack needs to know the name of the container its running in for the lambda functionality to work properly for us. In `docker-compose`, that requires knowing the project name. However, the only way to get access to that information in the compose file is if it's explicitly set by environment variable. Using the default or using `--project-name` doesn't propagate that information.)